src/reducer.js

```diff
@@ -37,10 +37,12 @@ const reducerMap = {
     ...state,
     [payload.name]: waiter(state[payload.name], { type: t.CLEAR, payload }),
   }),
-  [t.DESTROY]: (state, payload) => ({
-    ...state,
-    [payload.name]: waiter(state[payload.name], { type: t.RESET, payload }),
```

Comment:
This looked to be dispatching `RESET` from the `waiter-reducer`, but there isn't a `RESET` action as best as I can tell 😄

Reply:
Right, could remove destroy from the waiter-reducer as well

Reply:
Updated

Reply:
And if you want to drop the RESET_ALL above, it's the same as DESTROY_ALL and not documented, we can get it all cleaned up

Reply:
Nice, didn't catch that. Updated.